hearth/lib/hearth/http/header_list_parser.rb

```diff
@@ -25,6 +33,14 @@ def parse_http_date_list(value)
           value.split(',').each_slice(2).map { |v| Time.parse(v[0] + v[1]) }
         end
 
+        def parse_date_time_list(value)
+          value.split(',').map { |v| Time.parse(v) }
+        end
+
+        def parse_epoch_seconds_list(value)
+          value.split(',').map { |v| Time.at(v.to_i) }
```

Comment:
I think this should probably be a `to_f` instead of a `to_i` since these are defined as supporting millisecond precision everywhere I can find, and to_f should give the same values for any integer anyway so should be safe.

Reply:
Leaving at `.to_i` per discussion outlined here: https://github.com/smithy-lang/smithy-ruby/pull/191#discussion_r1581408052